lib/providers/transit.js

```diff
@@ -79,6 +79,19 @@ class TransitProvider {
         return response;
       });
   }
+
+  /**
+   * Get a unique ID for the given token and secret
+   *
+   * @param {String} token
+   * @param {Object} secret
+   * @param {Object} secret.key
+   * @param {Object} secret.ciphertext
+   * @return {String}
+   */
+  static getSecretID(token, secret) {
```

Comment:
This is a stop gap to allow the `TransitProvider` to define its own caching key for secrets, because Tokend expects that every secret returned by Vault has a lease and is cacheable.

Right now, Tokend doesn't understand the `renewable: false` flag that Vault returns. Once #67 lands, that'll change and I'll be able to yank this out. Here's [a work in progress diff](https://github.com/rapid7/tokend/compare/ca5e8de9b8951c18c1f77b23d8d8d259dd6551f6...41f11e94a029c6c331d47b09752db799b729ee12) of where that's headed.